bin/pulsar

```diff
@@ -151,6 +154,10 @@ if [ -z "$PULSAR_GLOBAL_ZK_CONF" ]; then
     PULSAR_GLOBAL_ZK_CONF=$DEFAULT_GLOBAL_ZK_CONF
 fi
 
+if [ -z "$PULSAR_CONFIGURATION_STORE_CONF" ]; then
+    PULSAR_CONFIGURATION_STORE_CONF=$DEFAULT_CONFIGURATION_STORE_CONF
+fi
+
```

Comment:
for BC consideration, if `-z "$PULSAR_GLOBAL_ZK_CONF"`, you might want to take the environment since this environment is passing from users.

basically

```
 if [ -z "$PULSAR_GLOBAL_ZK_CONF" ]; then
      PULSAR_GLOBAL_ZK_CONF=$DEFAULT_GLOBAL_ZK_CONF		      
      PULSAR_CONFIGURATION_STORE_CONF=$DEFAULT_GLOBAL_ZK_CONF
 fi
  		  
 if [ -z "$PULSAR_CONFIGURATION_STORE_CONF" ]; then
     PULSAR_CONFIGURATION_STORE_CONF=$DEFAULT_CONFIGURATION_STORE_CONF
 fi
```